### ISSUE TITLE
Names based on headers not working

### ISSUE THREAD

**Issue by dualhammers** [dvogt23/book-summary#18]
I am trying to use the `-m` command to name my entries based on their headers. I interpret header to me the top line with a single `#` indicating the highest level heading.

When set this option, however, the output does not seem to change.

**Comment by dvogt23** [comment]
yeah this is currently only an idea that has not yet been implemented. Sorry for that. Feel free to create a PR
